src/components/Person/Person.jsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './Person.scss';
+
+export const Person = ({
+  name,
+  age,
+  sex,
+  isMarried,
+  partnerName,
+}) => {
+  const partnerCheck = sex === 'm' ? 'wife' : 'husband';
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">
+        {`My name is ${name}`}
+      </h2>
+      {age
+        && (
+          <p className="Person__age">
+            {`I am ${age}`}
+          </p>
+        )}
+      <p className="Person__partner">
+        {isMarried
+          ? `My ${partnerCheck}'s name is ${partnerName}`
+          : 'I am not married'}
+      </p>
+    </section>
+  );
+};
+
+Person.defaultProps = {
+  age: '',
+  partnerName: '',
```

Comment:
if your isMarried = true and partnerName = '', you will see:
'My wife's name is '